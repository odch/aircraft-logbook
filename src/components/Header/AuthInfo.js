import React from 'react'
import PropTypes from 'prop-types'

const AuthInfo = props => <div data-cy="username">{props.username}</div>

AuthInfo.propTypes = {
  username: PropTypes.string.isRequired
}

export default AuthInfo
