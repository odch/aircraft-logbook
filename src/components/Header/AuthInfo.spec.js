import React from 'react'
import renderer from 'react-test-renderer'
import AuthInfo from './AuthInfo'

describe('components', () => {
  describe('Header', () => {
    describe('AuthInfo', () => {
      it('renders correctly', () => {
        const tree = renderer
          .create(<AuthInfo username="test@example.com" />)
          .toJSON()
        expect(tree).toMatchSnapshot()
      })
    })
  })
})
