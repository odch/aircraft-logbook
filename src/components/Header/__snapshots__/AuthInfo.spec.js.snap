// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components Header AuthInfo renders correctly 1`] = `
<div
  data-cy="username"
>
  test@example.com
</div>
`;
